examples/platforms/cc2538/radio.c

```diff
@@ -813,13 +826,19 @@ void cc2538RadioProcess(otInstance *aInstance)
 
 #if OPENTHREAD_CONFIG_CC2538_USE_RADIO_RX_INTERRUPT
     // Turn the receive interrupt handler back on now the buffer is clear.
-    HWREG(RFCORE_XREG_RFIRQM0) |= RFCORE_XREG_RFIRQM0_RXPKTDONE;
+    HWREG(RFCORE_XREG_RFIRQM0) |= CC2538_ALL_USED_INTERRUPTS_MASK;
 #endif
 }
 
 void RFCoreRxTxIntHandler(void)
 {
 #if OPENTHREAD_CONFIG_CC2538_USE_RADIO_RX_INTERRUPT
+
+    if (HWREG(RFCORE_SFR_RFIRQF0) & RFCORE_SFR_RFIRQF0_SRC_MATCH_FOUND)
+    {
+        sSrcMatchFound = true; // the next message received will have frame pending set
```

Comment:
Is it guaranteed the frame will be received and trigger the rx done interrupt? Will Rx done be triggered when FCS is incorrect? Maybe we should set clear the flag in rx_frame_accepted context.

Reply:
`sSrcMatchFound` is set to false even when crc check failed

Reply:
`RXPKTDONE A complete frame has been received.`
Currently, `readFrame` is triggered in the so called `rx_frame_accepted` context

Reply:
According to cc2538 user guide, the order or interrupts is:
 - When source matched: `RX_FRM_ACCEPTED -> SRC_MATCH_FOUND -> RXPKTDONE`
 - When source not matched: `RX_FRM_ACCEPTED -> RXPKTDONE`

So, it is guaranteed that there will always be a `RXPKTDONE ` after `SRC_MATCH_FOUND`. 
For messages filtered by frame filtering, none of `RX_FRM_ACCEPTED -> SRC_MATCH_FOUND -> RXPKTDONE` will be triggered.